src/binaries.py

```diff
@@ -54,6 +54,8 @@ def sort_firmwares(version: str) -> tuple:
 
 def explore_bridges() -> None:
     BRIDGES.append("2.0.31")
+    BRIDGES.append("2.0.31-arm64")
```

Comment:
At each current situation, could we even use all those bridges? We could filter them on the container-environment being ARM vs nonARM and only append the relevant ones. (Connected with not showing all of them in the HTML dashboard)

Reply:
We cannot use arm on non arm and vice versa Would be nice to only have the emulators we need for that system

Reply:
Moving to https://github.com/trezor/trezor-user-env/issues/147.